src/act_cell.py

```diff
@@ -38,14 +38,19 @@ def state_size(self):
 
     def __call__(self, inputs, state, timestep=0, scope=None):
 
+        self.state_is_tuple = False
```

Comment:
I haven't been able to find any "nice" way to extract the `self.cell`'s `state_is_tuple` field, it seems to be an internal field. But it should be possible to do:
`self.cell._state_is_tuple`. Is this preferred?